src/components/directorybrowser/directorybrowser.js

```diff
@@ -251,7 +251,7 @@ class DirectoryBrowser {
         }
         Promise.all([getSystemInfo(), getDefaultPath(options)]).then(
             responses => {
-                const systemInfo = responses[0];
+                systemInfo = responses[0];
```

Comment:
Same here: Overwriting the variable in the outer scope does not appear to be the desired behavior here.